src/components/DataControl/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+import { DataControlProps } from '../../hooks/useGraphql/types';
+import { useGraphql } from '../../hooks/useGraphql/useGraphql';
+
+const DataControl = <T,>({ ...props }: DataControlProps<T>) => {
+  useGraphql({ ...props });
+
+  useEffect(() => {
+    console.log('INSIDE');
```

Comment:
Something that will be replaced in the future?

Reply:
No, i just forgot to remove that 😄